tools/accuracy_checker/accuracy_checker/annotation_converters/README.md

```diff
@@ -103,13 +104,15 @@ Accuracy Checker supports following list of annotation converters and specific f
   * `dataset_meta_file` - path path to json file with dataset meta (e.g. label_map, color_encoding).Optional, more details in [Customizing dataset meta](#customizing-dataset-meta) section.
   * `semantic_only` - converts MS COCO dataset annotation to `SegmentationAnnotation`. (Optional, default value is False)
   * `masks_dir` - path to store segmentation masks in `semantic_only` mode
+  * `use_special_id_to_label_map` - allows to use special ID to label mapping (used for mapping subset of Pacsal VOC classes). Optional, default is False.
 * `mscoco_mask_rcnn` - converts MS COCO dataset to `ContainerAnnotation` with `DetectionAnnotation` and `CocoInstanceSegmentationAnnotation` named `detection_annotation` and `segmentation_annotation` respectively.
   * `annotation_file` - path to annotation file in json format.
   * `has_background` - allows convert dataset with/without adding background_label. Accepted values are True or False. (default is False).
   * `use_full_label_map` - allows to use original label map (with 91 object categories) from paper instead public available(80 categories).
   * `sort_annotations` - allows to save annotations in a specific order: ascending order of image id or ascending order of image size.
   * `sort_key` - key by which annotations will be sorted(supported keys are `image_id` and `image_size`, default is `image_id`).
   * `dataset_meta_file` - path path to json file with dataset meta (e.g. label_map, color_encoding).Optional, more details in [Customizing dataset meta](#customizing-dataset-meta) section.
+  * `use_special_id_to_label_map` - allows to use special ID to label mapping (used for mapping subset of Pacsal VOC classes). Optional, default is False.
```

Comment:
I think parameter name should be more clear  and contains info that it is connected with VOC